src/api-v1.js

```diff
@@ -665,13 +666,17 @@ api.declare({
       token: token,
     });
 
+    let workerId = slugid.v4();
+    let workerGroup = 'ec2_' + secret.region;
+
     return res.reply({
       data: secret.secrets,
-      scopes: secret.scopes,
+      workerId: workerId,
+      workerGroup: workerGroup,
       credentials: taskcluster.createTemporaryCredentials({
         scopes: [
           `assume:worker-type:${this.provisionerId}/${secret.workerType}`,
-          'assume:worker-id:*',
+          `assume:worker-id:${workerGroup}:${workerId}`,
```

Comment:
I think this is a legacy scope...

See queue sources it's in the comments of the API definition...
The new line is queue:worker-id:... If I recall correctly..